src/grid/atomgrid.py

```diff
@@ -99,21 +103,15 @@ def __init__(
         #     self._points = np.dot(self._points, rot_mt)
```

Comment:
Can these comments be removed? I believe rotation is taken care of in `_generate_atomic_grid` method.